src/linear_algebra/finite_dimensional.lean

```diff
@@ -850,6 +865,46 @@ by { unfold findim, simp [dim_top] }
 
 end top
 
+lemma findim_zero_iff_forall_zero [finite_dimensional K V] :
+  findim K V = 0 ↔ ∀ x : V, x = 0 :=
+begin
+  split; intro h,
+  { haveI := findim_eq_zero_iff_subsingleton.mp h,
+    exact λ x, subsingleton.elim x _ },
+  { rw findim_eq_zero_iff_subsingleton,
+    exact subsingleton_of_forall_eq _ h },
```

Comment:
I'm realizing that I probably should have also suggested `subsingleton_iff_forall_eq` which would make this proof `findim_eq_zero_iff_subsingleton.trans (subsingleton_iff_forall_eq 0)`, and do the same for `dim_zero_iff_subsingleton `

Reply:
Right, I will make another pr for this

Reply:
Alright, #6379